src/modifiers.jsx

```diff
@@ -25,7 +24,8 @@ var Modifiers = React.createClass({
     var list = [];
     var onChange = this.props.onChange;
     
-    _.each(this.props.modifiers, function(hex, index){
+    Object.keys(this.props.modifiers).forEach(function(index){
+      var hex = this.props.modifiers[index];
```

Comment:
You should pass the context `this` to `forEach` if you want to use it. Or keep a variable outside of the function to access the modifiers.